src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -586,6 +586,8 @@ var adyenHelperObj = {
         case 'Master':
         case 'MasterCard':
         case 'Mastercard':
+        case 'Master Card':
+        case 'Master card':
```

Comment:
What if we clean `cardType` first by downcasing it and removing all spaces? That would leave us with a single case (i.e. `mastercard`). Of course the other cased would need adjusting as well.